builder/builder-module.c

```diff
@@ -1520,18 +1550,41 @@ builder_module_build (BuilderModule  *self,
 
   /* Build and install */
 
-  if (meson || cmake_ninja)
-    make_cmd = "ninja";
+  if (simple)
+    {
+      guint length = g_strv_length (self->build_commands);
+      gint argc;
+      g_auto(GStrv) argv = NULL;
+      g_print ("Building with simple builsystem: %d build commands\n", length);
+
+      for (guint i = 0; i < length; i++)
+        {
+          g_print ("%s\n", self->build_commands[i]);
+
+          if (!g_shell_parse_argv (self->build_commands[i], &argc, &argv, error))
```

Comment:
We don't want to do magic parsing like this, run and *actual* shell, just like the later code that handles post_install